opencti-platform/opencti-front/src/utils/Entity.ts

```diff
@@ -1,10 +1,8 @@
-export const resolveLink = (type: string): string | null => {
+export const resolveLink = (type = 'unknown'): string | null => {
   switch (type) {
     case 'Dashboard':
-    case 'dashboard':
```

Comment:
I'm not sure we should remove this (and 'investigation'), I guess they have been added for a reason

Reply:
it was added here : https://github.com/OpenCTI-Platform/opencti/commit/dab14295b98dcc516d00e3295def4ee97c89676d#diff-93d809f17b979aa809952ecbb1617170d9b409015c6881c6fac76920340b83c3

Reply:
Oh... yes I see. We use resolveLink in the context of the workspace too. Seems like a side-effect of using same enum words 'dashboard' and 'investigation' for the workspace type. We just use the same function for both cases (entity type / workspace type). It's a bit misleading.

I'll reintroduce the lower-cased version with a proper comment.

Reply:
Good catch, thanks !